morris/roster.json

```diff
@@ -39,5 +39,8 @@
 		{
 			"name": "Maximilian Hartley"
 		}
+		{
```

Comment:
We'll need to make sure there is a comma between each object! Please could you add one between yours and Max's?

Reply:
and ive pushed and committed it